canonical-web-and-design/vanilla-framework
# Add buttons accessibility notes

## Done

- Add buttons accessibility notes. copy doc [here](https://docs.google.com/document/d/11H6xPtBRxB1a8KCmK7XFH1FeZDduQhSJXxq-wNJ_CZ8/edit#heading=h.qxwy2zql3yy7)

Fixes #4039 

## QA

- Open [demo](insert-demo-url)
- Check button accessibility section


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
